tools/datasets/kitti/download_kitti.sh

```
@@ -0,0 +1,54 @@
+    esac
+done
+
+readonly -a urls=(
+    "https://s3.eu-central-1.amazonaws.com/avg-kitti/data_odometry_gray.zip"
+    "https://s3.eu-central-1.amazonaws.com/avg-kitti/data_odometry_calib.zip"
+    "https://s3.eu-central-1.amazonaws.com/avg-kitti/data_odometry_poses.zip"
+)
+
+download_file() {
+    local url="$1"
+    local dest="${out_dir}/$(basename -- "${url}")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)